site/docs/tutorials/snake/publish-your-game.md

```diff
@@ -135,28 +135,7 @@ At the top you will find some "actions" you can take: `Watch/Unwatch`, `Star/Uns
 
 ## Uploading Your Game
 
-Once you've forked the repo, visit your version and navigate to the folder `site/static/carts/`. There you can upload your cart.wasm and a 160x160 cover-image of the game. A screenshot is an easy way. Simply press F9 while your game is running.
-
-Also: Rename your cart.wasm to something meaningful. Like `snake-spartan-edition.wasm`. Your screenshot should have the same name (but a different extension like `snake-spartan-edition.png`).
-
-Commit the changes and you're done with this folder.
-
-## Changing the List of Games
-
-Now that your game and the cover-image are uploaded, it's time to add them to the list of games. Navigate to `site/` and open the file `cart.js`.
-
-Press the `Edit` button (a pen) at the top. Now you can edit the file directly in the browser. Copy and Paste one of the entries and change the values to fit your specifications.
-
-Here is what each property is used for:
-
-| Property | Description |
-|----------|-------------|
-| slug     | Displayed in the address bar. A `lakeshooter` will result in this: https://wasm4.org/play/lakeshooter a `pong` would result in https://wasm4.org/play/pong |
-| title    | This is the name displayed on the collection. |
-| author   | The name displayed by WASM-4.  |
-| github   | Your GitHub username. This is used to display your avatar on the cover-page. |
-
-Once you've done this, commit those changes too.
+Follow the [Distribution](https://wasm4.org/docs/guides/distribution/#publish-on-wasm4org) instructions from the earlier Guides section.
```

Comment:
```suggestion
Follow the [Distribution](../../guides/distribution/#publish-on-wasm4org) instructions from the earlier Guides section.
```

This would link to the right page when running on localhost, or if we ever change the domain.

Reply:
Sounds good to me